api/scholarqa/rag/retriever_base.py

```diff
@@ -53,7 +53,7 @@ def snippet_search(self, query: str, **filter_kwargs) -> List[Dict[str, Any]]:
                 res_map["title"] = paper["title"]
                 res_map["text"] = snippet["text"]
                 res_map["score"] = fields["score"]
-                res_map["section_title"] = snippet["snippetKind"] if snippet["snippetKind"] != "body" else fields.get(
+                res_map["section_title"] = snippet["snippetKind"] if snippet["snippetKind"] != "body" else snippet.get(
```

Comment:
wondering if we should include another field that's just the snippetKind? for consumers that care only about `is-this-from-the-abstract` and `is-this-from-the-body`? (so they don't have to have logic figuring out that something is from the body if the section title isn't 'abstract' or 'title')